include/materials/norsand.tcc

```diff
@@ -49,6 +48,8 @@ mpm::NorSand<Tdim>::NorSand(unsigned id, const Json& material_properties)
     m_dilation_ = material_properties.at("m_dilation").template get<double>();
     // Parameter for shear modulus
     m_modulus_ = material_properties.at("m_modulus").template get<double>();
+    // Default tolerance
+    tolerance_ = material_properties.at("tolerance").template get<double>();
```

Comment:
Could we have this as an optional argument?

Reply:
Yes